### PR TITLE
docs(config) dev server guide updates

### PR DESCRIPTION
**Pull request by EugeneHlushko**
- actualize parameters and types
- small text updates
- useful links added for quick nav
- related content linked
- updating formatting
- Fixes: #2610 

Useful for reviewing:
https://github.com/webpack/webpack-dev-server/blob/master/lib/options.json
https://github.com/webpack/webpack-dev-server/blob/master/lib/Server.js

### CODE REVIEW

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -651,17 +651,19 @@ __webpack.config.js__
...
 ```
 
 And via the CLI
 
 ```bash
-webpack-dev-server --open 'Google Chrome'
+webpack-dev-server --open 'Chrome'
 ```
 
+T> The browser application name is platform dependent. Don't hard code it in reusable modules. For example, `'Chrome'` is google chrome on macOS, `'google-chrome'` on Linux and `'chrome'` on Windows.
```
Comment (montogeek) [inline src/content/configuration/dev-server.md]: Please change "google chrome" to Google Chrome.

**Comment by EugeneHlushko** [comment]
@montogeek hi man, thx for checking it out! just updated 👍

File: src/content/configuration/dev-server.md [hunk 2]
```diff
@@ -21,9 +22,11 @@ T> Options that are compatible with [webpack-dev-middleware](https://github.com/
 
 `object`
 
-This set of options is picked up by [webpack-dev-server](https://github.com/webpack/webpack-dev-server) and can be used to change its behavior in various ways. Here's a simple example that gzips and serves everything from our `dist/` directory:
+This set of options is picked up by [webpack-dev-server](https://github.com/webpack/webpack-dev-server) and can be used to change its behavior in various ways. Here's a simple example that gzips and serves everything from our `dist/` directory in the project root:
 
-```js
+__webpack.config.js__
+
+```jsavascript
```
Comment (byzyk) [inline src/content/configuration/dev-server.md]: typo 🙃

Comment (EugeneHlushko) [reply src/content/configuration/dev-server.md]: my god :D

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**